samples/multicore/hello_world/remote/boards/nrf54h20dk_nrf54h20_cpuppr_xip.conf

```diff
@@ -0,0 +1 @@
+CONFIG_XIP=y
```

Comment:
this is no longer needed, right?

Reply:
the goal is to execute from ram, if it will fit and work

Reply:
DK has changes to default to XIP=n, and there's now a `/xip` board variant. This is not needed as there's a board variant that sets this.

Reply:
Note that this is HWMv1 territory, so for now this appears as a board named `nrf54h20dk_nrf54h20_cpuppr_xip`.

Reply:
> this is no longer needed, right?

removed